Tensile/KernelWriterAssembly.py

```diff
@@ -3904,6 +3909,22 @@ def graTileAssignment(self, kernel, tP):
       kStr += staticMultiply(vgpr(rReg), vgpr(rReg), strideTile, sgpr(tmpSgpr), \
         "1. N offset: nOffset = nIdx * nStride(%u)" % strideTile)
       # block offset (no code. assuming num1DBlocks == 1)
+      if num1DBlocks > 1:
```

Comment:
This part is basically ported from LraTileAssignment.py.
TileAssignment calculation for DirectToVgpr should be same as calculation for local read.